src/main/kotlin/racing/domain/Car.kt

```diff
@@ -1,12 +1,27 @@
 package racing.domain
 
-data class Car(private var _location: Location = Location()) {
+class Car(val name: CarName, private val _location: Location = Location()) {
+    var location: Location = this._location
```

Comment:
```suggestion
class Car(val name: CarName, location: Location = Location()) {
    var location: Location = location
```
`_location`은 사용하지 않는 프로퍼티입니다. 단순히 생성자를 통해 넣어주고 값을 날려버리면 될 것 같아요.